v2/formattedoutput/formattedOutput.go

```diff
@@ -0,0 +1,159 @@
+package formattedoutput
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+	"sort"
+
+	"github.com/zmap/zlint/v2"
+	"github.com/zmap/zlint/v2/lint"
+
+)
+
+type resultsTable struct {
+	resultCount              map[lint.LintStatus]int
+	resultDetails            map[lint.LintStatus][]string
+	lintLevelsAboveThreshold map[int]lint.LintStatus
+	sortedLevels             []int
+}
+
+func (r resultsTable) newRT(threshold lint.LintStatus, results *zlint.ResultSet, longSummary bool) resultsTable {
```

Comment:
Doesn't technically matter in this case, since all of the member variables are effectively pointers, but this function should operate on pointer receivers.